src/App.js

```diff
@@ -1,11 +1,12 @@
+/*eslint-disable*/
+import React from 'react';
+import CalculatorApp from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Setting up React App at Microverse</h1>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <CalculatorApp />
+  </div>
+);
```

Comment:
- [ ] it would be better if you change this component from function to class-based as in the requirements
```JAVASCRIPT

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```